src/dev/build/tasks/optimize_task.js

```diff
@@ -48,6 +48,7 @@ export const OptimizeBuildTask = {
       cwd: build.resolvePath('.'),
       env: {
         FORCE_DLL_CREATION: 'true',
+        FORCE_WRITABLE_CACHE: 'true',
```

Comment:
Do you think we can indicate what cache this applies to somehow? Caching is everywhere, so I think we need to be a little more specific with our naming.

Reply:
@spalger do you have any suggestion? This would apply to any loader we configure with cache-loader. Something like `FORCE_WRITABLE_LOADERS_CACHE` ?

Reply:
Focusing on the cache-loader aspect makes sense to me, I'd be good with `KBN_CACHE_LOADER_WRITABLE=true` too

Reply:
👍